integration-tests/goss/goss-shared.yaml

```diff
@@ -205,6 +205,11 @@ http:
       - "Foo: bar"
     headers: ["Content-Type: application/json"]
     body: ['"Foo": "bar"']
+  http://httpbin/headers:
```

Comment:
Yaml will merge this incorrectly with the one above it. since it's a map key the last one will win.

Perhaps just combine this one with the one above it, something like this:

```yaml
http:
  http://httpbin/headers:
    status: 200
    timeout: 60000
    request-headers:
      - "Foo: bar"
    headers: ["Content-Type: application/json"]
    body:
      - '"Foo": "bar"'
      - '"User-Agent": "goss/0.0.0"'
```

Reply:
I did do lower case user agent because that is how it is in the code